project.py

```diff
@@ -259,6 +284,11 @@ def _add_relationships(self, item):
         except KeyError:
             print('2. KeyError at ' + item.key.text)
 
+        for customfield in item.customfields.findall('customfield'):
+            if customfield.get('key') == 'com.pyxis.greenhopper.jira:gh-epic-link':
```

Comment:
I'm curious, do you have example of such Jira issue(s)? What are they for?

Reply:
this is the type of custom field epic links are contained in, all of these issues: https://github.com/timja/jenkins-gh-issues-poc-06-18/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aepic